sklearn/utils/testing.py

```diff
@@ -68,12 +69,184 @@
            "assert_approx_equal", "SkipTest"]
 
 
+def _is_subtype(expected, basetype):
```

Comment:
Maybe it would be better to put this in a separate file like ``_unittest_backport.py`` or something?